specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/runCommands.json

```diff
@@ -0,0 +1,461 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RunCommandsClient",
+    "description": "The Run Commands Client.",
+    "version": "2018-06-01"
```

Comment:
Confirm that this file should be the same as 
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/runCommands.json
and only the version number should be updated.